### PR TITLE
Portfolio

### PR DESCRIPTION
**Pull request by AbdifatahYasin1**
# Portfolio Page

This is portfolio page designed by Abdifatah

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+        </div>
+
+        <button type="button" class="big-button">See project</button>
+      </div>
+      <!---about my self-->
+      <div class="workd-card about-myself">
```
Comment (Qoosim) [inline index.html]: - [ ] Kindly make this `section` tag instead of ordinary `div` for semantic reasons.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+        <h1>About Myself</h1>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <p id="text-color">LET’S CONNECT</p>
```
Comment (Qoosim) [inline index.html]: - [ ] Kindly make this `hx` tag. It could be `h2, h3, h4, ....` instead of `p` tag for sematic reason.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+
+        <ul>
+          <li><i class="fab fa-twitter"></i></li>
+        <li><i class="fab fa-linkedin"></i></li>
+        <li><i class="fab fa-medium"></i></li>
+        <li><i class="fab fa-github"></i></li>
+        <li><i class="fab fa-angellist"></i></li>
+
```
Comment (Qoosim) [inline index.html]: - [ ] It is highly recommended to make social icon as a link for sematic reason and accessibility purpose. Kindly use `<li><a>Your icon</a></li>` for all your social icons.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+  margin-bottom: 30px;
+}
+
+#text-color {
+  color: #396df2;
+}
+
+.below-button {
+  width: 40%;
+}
```
Comment (Qoosim) [inline style.css:223]: - [ ] Kindly give your button for `Get my resume` the button styling like Active, Hover and Pressed (see design). This is highly recommended for semantic reason.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,242 @@
...
+          <i class="fa fa-arrow-right"></i>
+        </h4>
+        <img src="images/SeparatorBottom.png" alt="" />
+
+        <h4 class="down-arrow">
+          Skills
+          <i class="fa fa-arrow-right"></i>
+        </h4>
+        <img src="images/SeparatorBottom.png" alt="" />
+      </div>
```
Comment (Qoosim) [inline index.html]: - [ ] All the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`. Kindly look into this and refactor your code.

**Review by Qoosim — CHANGES_REQUESTED** [review]
#   *Status: Required Changes* :recycle: 

Hi @AbdifatahYasin1, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

-   No linter errors. :heavy_check_mark:
-   Good README file. :heavy_check_mark: 
-   Correct GitHub Flow. :heavy_check_mark: 
-   meaningful commit messages. :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by franclobo — COMMENTED** [review]
Hi @AbdifatahYasin1, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't correspond with the activity in the dashboard, it appears as "Portfolio: finish mobile version", but it is about the first part of the portfolio.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Ugboaja-Uchechi — APPROVED** [review]
Hi @AbdifatahYasin1 , 

## Status: APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 
- Great job with your design and functionalities 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — COMMENTED** [review]
Hi @AbdifatahYasin1, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ⚠️

   - You have submitted a project with an old pull request. Each project should have a separate Pull Request.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

>Invalid Code Review Request **does not count** into the code reviews limit.

**Pull request merged**